oauth2/oauth2-jose/src/test/java/org/springframework/security/oauth2/jwt/MappedJwtClaimSetConverterTests.java

```diff
@@ -125,6 +125,15 @@ public void convertWhenUsingCustomConverterThenAllOtherDefaultsAreStillUsed() {
 
 	@Test
 	public void convertWhenConverterReturnsNullThenClaimIsRemoved() {
```

Comment:
Will you please add to the beginning of this test a comment tying it to the related GitHub issue?

For example:

```
// gh-10135
@Test 
public ...
```